### PR TITLE
Add login feature

### PR DESCRIPTION
**Pull request by qazillago**
Some description...

### CODE REVIEW

File: login.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+logpass
```
Comment (illia-user) [inline login.txt:1]: You must capitalize L

**Review by illia-user — CHANGES_REQUESTED** [review]
Please make these changes

**Comment by qazillago** [comment]
What's next?

**Comment by illia-user** [comment]
Nothing

**Review by illia-user — APPROVED** [review]
ok

**Review by illia-user — APPROVED** [review]

**Pull request merged**